docs/diagrams/LogicClassDiagram.puml

```diff
@@ -8,7 +8,9 @@ package Logic as LogicPackage {
 
 package "Parser Classes" as ParserClasses{
 }
-Class XYZCommand
+Class XYZCommand {
+    - String: dsa
```

Comment:
Can I check what is the purpose of this?

Reply:
My apologies, I just left that there while experimenting with the syntax and forgot to remove it later on.